map-utils.js

```diff
@@ -1,126 +1,146 @@
-return adjustedMapCenter      
-}
+  var mapCenter = options.mapCenter;
+  var mapZoom = options.mapZoom;
+  var offset = options.offset;
+  var reverse = options.reverse;
+  var directionalOffset = reverse ? offset * -1 : offset;
+  var z = Math.pow(2, mapZoom);
+  alert('z: ', z);
+  var point = getPointFromLatLng({
+    lat: (mapCenter, 'lat'),
+    lng: (mapCenter, 'lng')
+  }, { asPlainObject: true });
+  var adjustedMapCenter = getLatLngFromPoint({
+    x: point.x,
+    y: point.y + (directionalOffset / 2) / z
+  });
+  adjustedMapCenter.lat = adjustedMapCenter.lat.toFixed(6);
+  adjustedMapCenter.lng = adjustedMapCenter.lng.toFixed(6);
+
+  return adjustedMapCenter;
+};
 var latLngAreDifferent = function (firstLoc, secondLoc) {
-var distance;
-var firstLat = parseFloat((firstLoc.lat || firstLoc.latitude).toFixed(6))
-var firstLng = parseFloat((firstLoc.lng || firstLoc.longitude).toFixed(6))
-var secondLat = parseFloat((secondLoc.lat || secondLoc.latitude).toFixed(6))
-var secondLng = parseFloat((secondLoc.lng || secondLoc.longitude).toFixed(6))
-if (firstLat != secondLat || firstLng != secondLng) {
-return true
-}
-return false
-}
+  //var distance;  //commenting out because it's not used
+  var firstLat = parseFloat((firstLoc.lat || firstLoc.latitude).toFixed(6));
+  var firstLng = parseFloat((firstLoc.lng || firstLoc.longitude).toFixed(6));
+  var secondLat = parseFloat((secondLoc.lat || secondLoc.latitude).toFixed(6));
+  var secondLng = parseFloat((secondLoc.lng || secondLoc.longitude).toFixed(6));
+  if (firstLat !== secondLat || firstLng !== secondLng) {
+    return true;
+  }
+
+  return false;
+};
 var getAdjustedPositionFromMapInstance = function (instance, offset) {
-var zoomLevel = instance.getZoom()
-var center = instance.getCenter()
-var wrappedCenter = new google.maps.LatLng({
-lat: center.lat(),
-lng: center.lng()
-})
-var response = getAdjustedMapCenter({
-mapCenter: {
-lat: wrappedCenter.lat(),
-lng: wrappedCenter.lng() 
-},
-mapZoom: zoomLevel,
-offset: offset,
-reverse: true
-})
-response.zoomLevel = zoomLevel
-return response
-}
-module.exports = { 
-getAdjustedPositionFromMapInstance: getAdjustedPositionFromMapInstance,
-getDistance: getDistance, 
-degreesToRadians: degreesToradians,
-getRadiusLatLng: getRadiusLatlng,
-getPointFromLatLng: getPointFromLatLng,
-getLatLngFromPoint: getLatLngFromPoint,
-getFormattedPlaceName: getFormattedPlaceName,
-getAdjustedMapCenter: getAdjustedMapCenter,
-latLngAreDifferent: latLngAreDifferent
-}
+  var zoomLevel = instance.getZoom();
+  var center = instance.getCenter();
+  var wrappedCenter = new google.maps.LatLng({
+    lat: center.lat(),
+    lng: center.lng()
+  });
+  var response = getAdjustedMapCenter({
+    mapCenter: {
+      lat: wrappedCenter.lat(),
+      lng: wrappedCenter.lng()
+    },
+    mapZoom: zoomLevel,
+    offset: offset,
+    reverse: true
+  });
+  response.zoomLevel = zoomLevel;
+
+  return response;
+};
+
+var module = {};
```

Comment:
This works. Since we haven't introduced node modules yet, it wasn't necessary to address this as one. Another option would be to add "module" to the "globals" block in your eslint configuration file.